x-pack/docs/en/security/authorization/alias-privileges.asciidoc

```diff
@@ -9,10 +9,9 @@
 ==== Data stream privileges
 
 // tag::data-stream-security[]
-
-Use <<privileges-list-indices,indices privileges>> to control access to
-a data stream. Any role or user granted privileges to a data
-stream are automatically granted the same privileges to its backing indices.
+Use <<privileges-list-indices,index privileges>> to control access to a data
+stream. Any role or user granted privileges to a data stream are automatically
```

Comment:
I am aware this is existing sentence, but I don't quite follow it, especially the "Any role or user granted prvileges" part. I think it means granting any privileges for a datastream cascades down to its backing indices. But it's unclear to me.

Reply:
Thanks for raising this. Your understanding is correct. I reworded this to:

> Granting privileges for a data stream grants the same privileges for its backing indices.